### ISSUE TITLE
favicon from closed tab comes up on new website tab?

### ISSUE THREAD

**Issue by patrick99e99** [brave/browser-laptop#10176]
- Did you search for similar issues before submitting this one?
YES

- Describe the issue you encountered:

I was visiting a webpage:
https://investorshub.advfn.com/boards/read_msg.aspx?message_id=133353270

I had my my mouse on the "next" link on the right side of the page (under the search bar), and was trying to click it to move to the next message in the thread.  After the webpage finished loading, the content jumps---  so when I clicked "next", the content had shifted up causing me to click on "E*Trade" instead of the Next link.

A new window popped opened up with etrade..  Once I saw I was on E*Trade, I closed the window, as it was not my intention to go there...

A few minutes later, I clicked a different link from a slack channel, and I noticed that the Favicon for that website had somehow become E*Trade's favicon!!!

- Platform (Win7, 8, 10? macOS? Linux distro?):
Mac OS X 10.10.5 (14F2411)

- Brave Version (revision SHA):
0.17.19

- Steps to reproduce:
    cannot seem to reproduce
    
- Actual result:
Wrong favicon!

- Expected result:
Correct favicon!

- Will the steps above reproduce in a fresh profile? If not what other info can be added?
N/A

- Is this an issue in the currently released version?
Yes

- Can this issue be consistently reproduced?
Unfortunately I can't seem to make it happen again...

- Extra QA steps:
N/A

- Screenshot if needed:
<img width="1299" alt="screen shot 2017-07-27 at 11 21 38 am" src="https://user-images.githubusercontent.com/28946/28685797-cdb22d70-72bd-11e7-8403-1e2eafc988a3.png">

- Any related issues:
Note that in the screenshot, I am showing that I have two tabs of the same webpage open, the one on the right "[Integration] Runner - Asset" with the two orange/red chevrons has the correct favicon..  The one to the left has E*Trade's favicon.

**Comment by bsclifton** [comment]
Thanks for the report, @patrick99e99 😄  This issue is a duplicate of https://github.com/brave/browser-laptop/issues/1646

Since you have better steps, I'll update the original issue and give it a +1 on your behalf 😄

**Comment by bsclifton** [comment]
After re-reading, I was mistaken: this appears to be different (but related).  Re-opening

**Comment by LaurenWags** [comment]
Just saw this on 0.18.24. I had 2 pinned tabs (one of which was github.com) and about 25 tabs open. I clicked on the non-pinned tab next to my pinned tabs and started closing the tabs in quick succession (using the x icon on the tab, not cmd+w or right click to close tab). After closing about 12-13 tabs I noticed that my pinned tab was now displaying the wrong favicon:
![screen shot 2017-08-31 at 2 39 31 pm](https://user-images.githubusercontent.com/28145373/29939797-1ea53a52-8e5b-11e7-95e3-35547ffff46b.png)

**Comment by LaurenWags** [comment]
+1 from community: https://community.brave.com/t/tab-icon-changing-to-last-closed-tab/6103

**Comment by LaurenWags** [comment]
+1 from community https://community.brave.com/t/wrong-icon-on-last-tab-on-osx/21371

**Comment by LaurenWags** [comment]
Reproduced on 0.22.716 - had multiple tab pages open, Opened Preferences, enabled Metamask, after the metamask page opened, I closed it and Preferences got the MetaMask favicon:
![screen shot 2018-05-18 at 9 18 41 am](https://user-images.githubusercontent.com/28145373/40236762-93904be8-5a7c-11e8-87f5-fecd0de4f4e4.png)

**Comment by srirambv** [comment]
This looks similar to https://github.com/brave/browser-laptop/issues/6513 which has been an issue for a while.

**Comment by LaurenWags** [comment]
yep -  I definitely think https://github.com/brave/browser-laptop/issues/10176#issuecomment-390204464 is a case of #6513. thanks!
